utils/api/configurations.js

```diff
@@ -106,13 +106,15 @@ export const configureStatus = (status) => {
 export const configureMessages = (data) => {
   // NOTE(alishaevn): doing some basic filtering here until we come to a resolution on
   // https://github.com/assaydepot/scientist_api_v2/issues/167
-  const filteredMessages = data.filter(d => d.user_ref && d.body)
+  const filteredMessages = data.filter(d => d.user_ref && d.body && d.created_at && d.attachments)
```

Comment:
```suggestion
  const filteredMessages = data.filter(d => d.user_ref && d.body)
```

Reply:
all requests should have a `created_at` value and we don't want to require attachments

Reply:
also, once we get the `messages` endpoint back from the api, we won't need this filtering at all.